### DISCUSSION TITLE
New Maven Build in mvnrepository.com

### DISCUSSION THREAD

**Discussion by mntripathi** [gporter0205/schwab-api-client#6] [General]
Hi,
Can you please create new build? I can see only old build 0.1.1

https://mvnrepository.com/artifact/com.pangility/schwab-api-client

**Comment by gporter0205** [comment]
I'm not sure where mvnrepository.com is pulling it's info from, but I see the latest version when I search directly in maven central. https://central.sonatype.com/artifact/com.pangility/schwab-api-client
